src/features/layout/footer/Footer.tsx

```diff
@@ -7,7 +7,7 @@ const Footer = () => {
 
   return (
     <>
-      <footer className=" h-fit w-full bg-[#232323] text-white">
+      <footer className="w-full bg-[#232323] text-white">
```

Comment:
when I click on one of the footers links, the scroll seems to stay on the bottom. Check and see if maybe a (custom) useScrollToTop hook could be useful

Reply:
Good point. I think we will make a scroll button globally, but i will try to make a hook also.

Reply:
Yeah, scroll to top is a nice feature, but when you enter new page you expect to be on the top of it usually (unless explicitly not :) )

Reply:
Done.